scripts/pugme.js

```diff
@@ -0,0 +1,28 @@
+// Description:
+//   Pugme is the most important thing in life
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   randbot pug me - Receive a pug
+//   randbot pug bomb N - get N pugs (max 10)
+
+module.exports = function (robot) {
+  robot.respond(/pug me/i, (msg) => {
+    msg.http('http://pugme.herokuapp.com/random').get()((err, res, body) => {
+      msg.send(JSON.parse(body).pug);
+    });
+  });
+  robot.respond(/pug bomb( (\d+))?/i, (msg) => {
+    console.log(msg.match[2]);
+    const count = parseInt(msg.match[2], 10) > 10 ? 10 : msg.match[2] || 5;
```

Comment:
`parseInt` on the `msg.match[2]` at the end there just to be consistent with types. Right now, there is one case that it can be a `string`, otherwise it's a `Number`.